agents/go-agents/.gitignore

```diff
@@ -0,0 +1,2 @@
+src/main/go/exec-agent/exec-agent
```

Comment:
This is executable of go program. It may be created in development if user manually builds go sources. 
Is something wrong with this exclusion?